.changeset/late-melons-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Add Signal Icon
```

Comment:
```suggestion
Adds 1 new icon: `signal`
```

Tiny change as we always start with a present tense verb for changeset and for icons, we include the exact name users can use.